packages/paste-website/src/pages/index.tsx

```diff
@@ -1,10 +1,111 @@
 import * as React from 'react';
+import styled from '@emotion/styled';
+import {themeGet} from 'styled-system';
+import {Button} from '@twilio-paste/button';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
 import {SiteWrapper} from '../components/default-site-wrapper';
+import {Grid} from '../components/grid';
+import {StyledHr} from '../components/StyledHr';
+import {Brush} from '../components/icons/Brush';
+import {Terminal} from '../components/icons/Terminal';
+import {WindowLayout} from '../components/icons/WindowLayout';
+
+const LandingContainer = styled(Box)`
+  margin-right: auto;
+  margin-left: auto;
+`;
+
+const StyledButton = styled(Button)`
+  margin-right: ${themeGet('space.space40')};
+  text-decoration: none;
+`;
+
+const IconCircle = styled(Box)`
```

Comment:
I was mostly able to use the box component, but still needed the `justify-content` and `align-items` values to style this to match the comp. I also kept the `width` and `height` values here because the we're using our `size` tokens for those props, and we currently don't have a size token for `60px`.

Reply:
We need a flexbox component so bad right now